packages/scaffolding-cli/templates/src/csr/README.md

```diff
@@ -0,0 +1,49 @@
+# Client Side Rendering Web Application
+
+## Implementation details
+
+The template is a Client Side Rendered (CSR) implementation using:
+
+- [React.js](https://reactjs.org/)
+- [Jest](https://jestjs.io/)
+- [Cypress](https://docs.cypress.io/)
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br /> Open
```

Comment:
html

Reply:
Sorry this was just an edited version of default CRA readme.